drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -221,7 +221,9 @@ def to_pick_list_option
   def data_collection_features
     # Create OpenStruct for each enabled feature
     Hmis::Form::Definition::DATA_COLLECTION_FEATURE_ROLES.map do |role|
-      base_scope = Hmis::Form::Instance.with_role(role)
+      # To discuss: We don't currently support retiring forms, but if we do, this would break in a way that we might not catch right away.
+      # What's the most graceful way to future proof this? e.g. published_or_retired scope; reuse valid_status_for_submit?
```

Comment:
Good point, yes, I think we should include both published and retired to maintain the current intended behavior, and add a comment about why. "valid_status_for_submit" probably isn't the clearest since this isn't a submission.

https://github.com/open-path/Green-River/issues/6159 outlines switching this to always show a feature if there is any data for it, in addition to checking the form rules. May make sense to just update the comment to reference that ticket as well?